sklearn/ensemble/bagging.py

```diff
@@ -516,7 +517,8 @@ def predict(self, X):
         y : array of shape = [n_samples]
             The predicted classes.
         """
-        return self.classes_.take(np.argmax(self.predict_proba(X), axis=1),
+        predicted_probabilitiy = self.predict_proba(X)
```

Comment:
Without this change we need to add a `check_is_fitted(self, "classes_")` before the return... And I felt this was redundant since that is being already done in `predict_proba` method...